cmd/test-bot.rb

```diff
@@ -59,6 +59,10 @@ def test_bot_args
              description: "Only run the tap syntax check step."
       switch "--only-formulae",
              description: "Only run the formulae steps."
+      switch "--only-formulae-bottle",
+             description: "Only run the formulae bottle step."
+      switch "--only-formulae-dependents",
+             description: "Only run the formulae dependents step."
```

Comment:
1. It seems like these options should conflict with each other.
2. There will be at least three `--only-formulae...` switches. I think it makes sense to convert it to a flag.